### ISSUE TITLE
Give detail about GiftCard non application

### ISSUE THREAD

**Issue by Roshyo** [Setono/SyliusGiftCardPlugin#167]
It would be nice for the customer to have information about why he can not apply a gift card. Something like: 

* This Gift Card does not exist
* This Gift Card balance is 0
* This Gift Card can not be used on this channel
* ...

**Comment by loevgaard** [comment]
I agree. Right now you don't get any feedback if you don't input anything either:

https://user-images.githubusercontent.com/2412177/146510719-cd755710-d9e7-4c2e-82bc-e3b8438a3e0e.mov

or when you input something wrong:

https://user-images.githubusercontent.com/2412177/146510841-85a5bd94-c453-4caa-8cd6-b77cce117ce9.mov

**Comment by Roshyo** [comment]
I do have a feedback for empty or wrong GC code

![image](https://user-images.githubusercontent.com/9363039/148393486-fefdad51-306a-4429-92c0-05e3191cc073.png)
![image](https://user-images.githubusercontent.com/9363039/148393532-0ed47e51-463c-4aee-a094-710361bc55d8.png)

**Comment by loevgaard** [comment]
Weird, maybe I didn't clear my cache completely. Thanks

**Comment by Roshyo** [comment]
You can leave this opened as there is no message for specific: 

* This Gift Card balance is 0
* This Gift Card can not be used on this channel

**Comment by loevgaard** [comment]
Okay, will do :)
